### ISSUE TITLE
CrashBrowser: PushMediaSendJob

### ISSUE THREAD

**Issue by moxie0** [WhisperSystems/TextSecure#4099]
`````
java.lang.NullPointerException: Attempt to read from null array
	at org.thoughtcrime.securesms.jobs.PushMediaSendJob.deliver(PushMediaSendJob.java:108)
	at org.thoughtcrime.securesms.jobs.PushMediaSendJob.onSend(PushMediaSendJob.java:70)
	at org.thoughtcrime.securesms.jobs.SendJob.onRun(SendJob.java:40)
	at org.thoughtcrime.securesms.jobs.MasterSecretJob.onRun(MasterSecretJob.java:18)
	at org.whispersystems.jobqueue.JobConsumer.runJob(JobConsumer.java:76)
	at org.whispersystems.jobqueue.JobConsumer.run(JobConsumer.java:46)
`````

**Comment by mcginty** [comment]
uncertain root cause of the problem

**Comment by notleigh** [comment]
I encountered this crash today.  I received a MMS which hadn't downloaded, when I touched 'retry' to view the content, the application crashed with "Unfortunately, TextSecure has stopped."  Since then the app crashes on startup every time.

The user who sent the MMS is on Signal, but the message was a regular insecure MMS.

I've submitted a report through Google's feedback system, and in the crash details panel saw the crash occurred with a java.lang.NullPointerException in PushMediaSendJob.java on line 108.
